webservices/src/main/java/com/javaschool/webservices/controller/PackageController.java

```diff
@@ -0,0 +1,31 @@
+package com.javaschool.webservices.controller;
+
+import java.util.List;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.javaschool.webservices.service.PackageService;
+
+@RestController
+@RequestMapping("packageInformation")
+public class PackageController {	
+	
+	private PackageService packageService;
+	
+	public PackageController(PackageService packageService) {
+		this.packageService = packageService;
+	}
+		
+	@GetMapping("/getSizes")
```

Comment:
check the REST conventions https://restfulapi.net/resource-naming/ you should normally name the endpoint to represent the resource you are dealing with, with just one word, the verb is already present in the HTTP action, so you do not need to specify it here, and the endpoint should be named the same for other actions, like when updating a size.

Reply:
I change the way to use properties, now I created a @ConfigurationProperties class for rabbit configuration.

Reply:
rest service name changed following REST conventions.